internal/pkg/manifest/workload.go

```diff
@@ -55,8 +55,9 @@ type Workload struct {
 
 // Image represents the workload's container image.
 type Image struct {
-	Build    BuildArgsOrString `yaml:"build"`    // Build an image from a Dockerfile.
-	Location *string           `yaml:"location"` // Use an existing image instead.
+	Build        BuildArgsOrString `yaml:"build"`       // Build an image from a Dockerfile.
+	Location     *string           `yaml:"location"`    // Use an existing image instead.
+	DockerLabels map[string]string `yaml:"labels,flow"` // Apply Docker labels to the container at runtime.
```

Comment:
Can we add this field for the sidecar image as well?

Reply:
Yep! Done in latest commit.